src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/reported/PunishmentsService.kt

```diff
@@ -351,6 +353,27 @@ class PunishmentsService(
     }
   }
 
+  fun createPunishmentComment(
+    adjudicationNumber: Long,
+    punishmentComment: PunishmentCommentRequest,
+  ): ReportedAdjudicationDto {
+    val reportedAdjudication = findByAdjudicationNumber(adjudicationNumber).also {
+      it.validateCanAddPunishmentComment()
+    }
+
+    if (punishmentComment.comment.isBlank()) {
```

Comment:
good spot btw.  But lets not fix this here.  Raise a ticket for front end to trim all text before sending, as this issue is present across the system currently.  Happy for you to remove this check.

Reply:
check removed. ticket for front end:  https://dsdmoj.atlassian.net/browse/NN-5121